chroma_core/migrations/0016_action_queue.py

```diff
@@ -18,25 +18,17 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.RunSQL(sql=forward_lustre_fid, reverse_sql=backward_lustre_fid),
-        migrations.CreateModel(
-            name="ActionType",
-            fields=[
-                ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
-                ("name", models.CharField(max_length=64)),
-            ],
-            options={"ordering": ["id"],},
-        ),
         migrations.CreateModel(
             name="FidActionQueue",
             fields=[
                 ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
                 ("fid", chroma_core.models.action_queue.LustreFidField()),
-                ("entries", django.contrib.postgres.fields.jsonb.JSONField(default={})),
-                ("failed", models.PositiveSmallIntegerField()),
+                ("data", django.contrib.postgres.fields.jsonb.JSONField(default={})),
+                ("failed", models.PositiveSmallIntegerField(default=0)),
```

Comment:
This column name makes me think of a boolean, perhaps `return_code` would be better?